README.md

```diff
@@ -14,6 +12,21 @@ A mongodb database named lizardboard must be created prior to starting the appli
 - Ensure `mongo` is running
 - yarn install
 
+## HTTP API
+description                                  | CRUD   | verb   | path                                     |
+-------------------------------------------- | ------ | ------ | ---------------------------------------- |
+Show all users                               | read   | get    | /api/v1/users                            |
+Create new user                              | create | post   | /api/v1/users                            |
+Show user by ID                              | read   | get    | /api/v1/users/:id                        |
+Update user by ID                            | update | put    | /api/v1/users/:id                        |
+Delete user by ID                            | delete | delete | /api/v1/users/:id                        |
+Show all dashboards                          | read   | get    | /api/v1/dashboards                       |
+Create a dashboard                           | create | post   | /api/v1/dashboards                       |
```

Comment:
For the Create and Update endpoints, we need to somehow list the expected data and data types, so that users of the API know what they are expected to post/put/etc.